File: app/src/main/java/ru/melod1n/vk/api/model/VKLongPollServer.java
package ru.melod1n.vk.api.model;

import org.json.JSONObject;

import java.io.Serializable;

public class VKLongPollServer extends VKModel implements Serializable {
    private static final long serialVersionUID = 1L;

    public String key;
    public String server;
    public long ts;

    public VKLongPollServer(JSONObject source) {
        this.key = source.optString("key");
        this.server = source.optString("server").replace("\\", "");
        this.ts = source.optLong("ts");
    }
}

File: app/src/main/java/ru/melod1n/vk/api/model/VKLongPollServer.kt
package ru.melod1n.vk.api.model

import org.json.JSONObject
import java.io.Serializable

class VKLongPollServer(source: JSONObject) : VKModel(), Serializable {
    var key: String = source.optString("key")
    var server = source.optString("server").replace("\\", "")
    var ts: Long = source.optLong("ts")

    companion object {
        private const val serialVersionUID = 1L
    }

}